test/keyword.test.js

```diff
@@ -0,0 +1,115 @@
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const app = require('../server.js')
+
+chai.use(chaiHttp)
+
+
+function checkResponse(res) {
+  res.should.have.status(200)
+  res.body.should.be.an('array')
+  res.body.should.not.be.empty
+}
+
+function checkWord(keyword) {
+  keyword.should.have.property('word')
+  keyword.word.should.an('string')
+  keyword.word.should.not.be.empty
+}
+
+function checkVotes(keyword) {
+  keyword.should.have.property('votes')
+  keyword.votes.should.be.an('array')
+  keyword.votes.forEach((vote) => {
+    vote.should.be.an('object')
+    vote.should.have.property('time')
+    vote.should.have.property('sum')
+  })
+}
+
+function checkValidError(res) {
+  res.should.have.status(500)
+  res.body.should.not.be.empty
+  res.body.should.have.property('err')
+  res.body.should.have.property('givens')
+}
```

Comment:
In the article tests, these helper functions are after all of the describes. Where should we place them? @JackyChiu